src/CSF.SRDashboard.Client/Services/CommonService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Localization;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Services
+{
+    public class CommonService
+    {
+        private IHttpContextAccessor httpContextAccessor { get; set; }
+
+        public CommonService(IHttpContextAccessor httpContextAccessor)
+        {
+            this.httpContextAccessor = httpContextAccessor;
+        }
+
+        public string GetUserLanguage()
+        {
+
+            httpContextAccessor.HttpContext.Request.Cookies.TryGetValue(CookieRequestCultureProvider.DefaultCookieName, out string language);
+
+            if (string.IsNullOrWhiteSpace(language))
+            {
+                language = "c=en|uic=en";
+            }
+
+            language = language.Substring(2, 2);
+
+            return language;
+        }
+
+        public bool IsEnglish()
+        {
+            return GetUserLanguage().Equals("en", StringComparison.OrdinalIgnoreCase);
+        }
+
+        public string DateFormat(DateTime dateToFormat, string format, string language = null)
+        {
+
+            if (string.IsNullOrWhiteSpace(language))
+            {
+                language = GetUserLanguage();
+            }
+
+            CultureInfo culture = new CultureInfo("en-CA");
+
+            if (language.Equals("Fr", StringComparison.OrdinalIgnoreCase))
```

Comment:
Though technically it doesn't matter in which case we have "fr" I'd suggest for consistency's sake to have both letters in the same case. Also adding a read-only parameter "fr" to the class to avoid using magic strings.
```suggestion
            if (language.Equals("fr", StringComparison.OrdinalIgnoreCase))
```